docs/joss_paper/paper.md

```diff
@@ -55,67 +67,67 @@ bibliography: paper.bib
+The loss functions described in the previous sections are often combined with a deformation regularisation term on the predicted displacement field, in order to ensure the predicted deformation is smooth. The deformation can be regularised by L1-, L2 norms of the displacement gradient and bending energy, based on the first- or second derivatives of the predicted DDF. Primarily predicting a general dense displacement field (DDF) from the network (discussed further in the next section), other parameterised transformation models can be readily added to the DeepReg framework, such as rigid transformation model and free-form deformation model based on B-splines [@rueckert1999nonrigid], to further constrain the predicted transformation. As an example, DeepReg implements an affine transformation model with twelve degrees of freedom.
 
-The latest result from a different registration formulation challenged the use of deformation regularisation and consequently the necessity of predicting dense displacement field [@hu2019conditional], also implemented in `DeepReg`.
+The latest result from a different registration formulation challenged the use of deformation regularisation and consequently the necessity of predicting dense displacement field [@hu2019conditional], also implemented in DeepReg.
 
 # Implementation
 ## Registration network
-The network accepts two images as input: a fixed image, and a moving image. When available and specified, pairs of moving labels and/or fixed labels can also be the input of the network to compute the label dissimilarity loss. A dense displacement field (DDF) can be directly predicted by the network, integrated by a predicted dense static velocity field (SVF or DVF) using the scaling-squaring method [@ashburner2007fast; @vercauteren2009diffeomorphic], or computed from a predicted affine transformation. In the conditional segmentation algorithm, the network directly predicts the warped moving labels [@hu2019conditional].
+The network accepts two images as input: a fixed image, and a moving image. When available and specified, pairs of labels (here, a moving and a fixed labels are a label pair, annotated from the moving and fixed images, respectively) can also be the input of the network to compute the label dissimilarity loss. A DDF can be directly predicted by the network, integrated by a predicted dense static velocity field (SVF or DVF) using the scaling-squaring method [@ashburner2007fast; @vercauteren2009diffeomorphic], or computed from a predicted parameterised transformation such as the affine transformation model. In the conditional segmentation algorithm, the network directly predicts the warped moving labels [@hu2019conditional].
```

Comment:
"(here, a moving and a fixed labels are a label pair, annotated from the moving and fixed images, respectively)"
->
"(here, a moving label and a fixed label, annotated from the moving and fixed images, respectively)"